Html insure landing review

RockyStocky opened the pull request.

RockyStocky committed:
change tags

change tag "div" to"nav", because is more semantic to the element function

RockyStocky committed:
change navegation links

change the navegation links to real content

RockyStocky committed:
change name class

change the class "insurancePlans" name to "insuranceIntro"  in tag section.

RockyStocky committed:
add tag h2

RockyStocky committed:
insureDifferential update

add div tags to wrap each component

idmarjr commented on index.html:
```diff
@@ -11,32 +11,39 @@
+      <p>Our plans aren`t full of conditions and clauses to prevent payouts. We make sure you`re covered when you need.</p>
+    </div>  
+    </section>
 </main>
```
I think we're missing a section from the interface prototype.
Right after the last item at `.insuranceDiferential` there's a content block about how _we work_ that is missing in the code:

<img width="367" alt="Screen Shot 2020-08-15 at 9 11 23 PM" src="https://user-images.githubusercontent.com/301545/90319817-0c8fbd80-df3c-11ea-86c5-fa29c996c67c.png">

RockyStocky committed:
pageFooter changes

-change element div to tag ul in "oursocialMedia",
-move out enterprise logo from "oursocialMedia"

RockyStocky committed:
tag change pageFooter

change elemet <p> to <h4> to improve semantics

RockyStocky committed:
add section "moreaboutUs" in main

RockyStocky committed:
remove period
